plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -486,6 +481,38 @@ protected TableSnapshot getSnapshot(ConnectorSession session, SchemaTableName ta
         }
     }
 
+    private static long getVersionNumberFromConnectorTableVersion(TrinoFileSystem fileSystem, String tableLocation, ConnectorTableVersion version)
```

Comment:
version is always a number right? so let's call it `getVersion`

`fromConnectorTableVersion` is already defined by the method argument